tensorboard/plugins/hparams/get_experiment.py

```diff
@@ -38,7 +38,7 @@ def __init__(
             not request.HasField("include_metrics")
             or request.include_metrics
         )
-        self._hparams_limit = request.hparams_limit
+        self._hparams_limit = request.hparams_limit if request.HasField("hparams_limit") else 3
```

Comment:
Not sure this is what you had in mind, but `3` was the proto field number, rather than a default value defined as the limit. If the field is undefined (default value for int proto would be 0), it would be considered as "no limit", see the relevant call sites below:
- https://github.com/tensorflow/tensorboard/blob/d1ab6e7a39e4dc4d556a8a73c0ae5c1b116801ba/tensorboard/data/provider.py#L439
- https://github.com/tensorflow/tensorboard/blob/d1ab6e7a39e4dc4d556a8a73c0ae5c1b116801ba/tensorboard/plugins/hparams/backend_context.py#L90

But this is indeed a problem when the request isn't `GetExperimentRequest`.  Can you please update this statement to:
```
self._hparams_limit = (
        request.hparams_limit
        if isinstance(request, api_pb2.GetExperimentRequest)
        else None
    )
``` 
Please also update the docstring at line #30 to `request: A request proto.` accordingly.

Reply:
Done